src/foundation.ts

```diff
@@ -0,0 +1,10 @@
+import { LitElement } from 'lit';
+
+/** Constructor type for defining `LitElement` mixins. */
+export type LitElementConstructor = new (...args: any[]) => LitElement;
+
+export { newOpenDocEvent } from './foundation/open-doc.js';
+export type { OpenDocEvent } from './foundation/open-doc.js';
```

Comment:
Nice to know you have to specify further. Where did you get this info from?

Reply:
https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-8.html version 3.8 introduced the feature.